spec/unit/complex_multiple_example_spec.rb

```diff
@@ -39,18 +39,18 @@
   it 'should be able to wait into waiting if polite' do
     auth.left_suspend!
     expect(auth.may_left_wait?(:waiting, :please)).to be true
-    auth.left_wait!(nil, :please)
```

Comment:
@anilmaurya I noticed `nil` is explicitly included in argument when triggering event in the specs. Is this a common in actual usage? This is different from the usage examples written in https://github.com/aasm/aasm/wiki/State-transitions-with-guard-clauses-that-require-arguments